### PR TITLE
Update operations.md

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
Operations topic: https://neon-next-git-dprice-polling-operations-neondatabase.vercel.app/docs/manage/operations
- Added operations API examples
- Added topic about polling operation status:

Neon API topic: https://neon-next-git-dprice-polling-operations-neondatabase.vercel.app/docs/reference/api-reference#using-the-neon-api-reference-to-construct-and-execute-requests
- Added links to API examples in the doc
- Added link to content about polling operation status

### CODE REVIEW

File: content/docs/manage/operations.md [hunk 1]
```diff
@@ -4,34 +4,187 @@ enableTableOfContents: true
...
-- `delete_tenant`: Deletes the stored data when a Neon project is deleted.
-- `delete_timeline`: Deletes a branch. The terms _timeline_ and _branch_ are synonymous. The term _timeline_ is used internally in Neon.
-- `start_compute`: Starts a compute endpoint when there is an event or action that requires compute resources. For example, connecting to a suspended compute endpoint initiates this operation.
-- `suspend_compute`: Suspends a compute endpoint after a period of inactivity. For information about how Neon manages compute resources, see [Compute lifecycle](/docs/introduction/compute-lifecycle/).
+| Operation | Description  |
+|:---|:---|
+| `apply_config` | Applies a new configuration to a Neon object or resource. For example, creating, deleting, or updating  PostgreSQL users and databases initiates this operation. |
+| `check_availability`  | Checks the availability of a [compute endpoint](/docs/reference/glossary#compute-endpoint). This operation, performed by the [availability checker](/docs/reference/glossary#availability-checker), is a periodic load generated by the Control Plane that makes sure a compute endpoint is able to start and read and write data.  |
```
Comment (ololobus) [inline content/docs/manage/operations.md:12]: Checks availability of the data in the branch, as it's now primarily connected to a branch, not endpoint. Yet, it involves starting the endpoint. So we check that compute can start on some particular branch. We also skip branches without endpoint

File: content/docs/manage/operations.md [hunk 2]
```diff
@@ -4,34 +4,187 @@ enableTableOfContents: true
...
+| `create_compute`   | Creates a compute endpoint. Creating a compute endpoint, a branch with a compute endpoint, or a Neon project initiates this operation. A Neon project is created with a read-write endpoint by default. For related information, see [Manage compute endpoints](/docs/manage/endpoints).  |
```
Comment (ololobus) [inline content/docs/manage/operations.md:13]: There is no `create_compute` now, we can drop it here and from the spec

File: content/docs/manage/operations.md [hunk 3]
```diff
@@ -4,34 +4,187 @@ enableTableOfContents: true
...
 - **Date**: The date and time the operation occurred.
 
 ## View operations with the Neon API
 
-You can also view operations using the Neon API, which provides operations data in an unmodified format. For example:
+The following topics are covered in this section:
+
+- [List operations](#list-operations): Describes how to list all operations for a Neon project.
+- [List operations with pagination](#list-operations-with-pagination): Describes how to list all operations for a Neon project and paginate the response.
+- [List operation](#list-operation): Describes how to retrieve the details for a specific operation by the operation ID.
```
Comment (ololobus) [inline content/docs/manage/operations.md:40]: Get operation?

File: content/docs/manage/operations.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/manage/operations.md:12]: Revised.

File: content/docs/manage/operations.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/manage/operations.md:13]: Removed from this topic. I will remove it from the spec in another PR.

File: content/docs/manage/operations.md (hunk 3 above)
Comment (danieltprice) [reply content/docs/manage/operations.md:40]: Changed as suggested

**Comment by danieltprice** [comment]
@ololobus Thanks for your review. The suggested changes have been addressed.

**Review by ololobus — APPROVED** [review]
I haven’t read it thoroughly, but I think it’s ready to go

**Pull request merged**